client/web/admin/src/components/Apigw/Profiler/CProfilerRouteHits.vue

```diff
@@ -34,12 +34,11 @@
 
         <c-input-confirm
           :disabled="!items.length"
-          :borderless="false"
+          :processing="processingConfirm"
```

Comment:
I would avoid calling it processingConfirm, instead it should denote what its processing.
In this case its the purge, so that would mean `processingPurge`
Same in all other places